tests/test_kratos_outputs_io.py

```diff
@@ -0,0 +1,142 @@
+        )
+
+        # 2. Vtk (Paraview)
+        vtk_output_parameters1 = VtkOutputParameters(
+            file_format="binary",
+            output_precision=8,
+            output_interval=100.0,
+            nodal_results=nodal_results1,
+            gauss_point_results=gauss_point_results1,
+        )
+        vtk_output_parameters2 = VtkOutputParameters(
+            file_format="ascii",
+            output_precision=8,
+            output_control_type="step",
+            output_interval=100.0,
+            nodal_results=nodal_results2,
+            gauss_point_results=gauss_point_results2,
+        )
+        # 3. Json
+        json_output_parameters1 = JsonOutputParameters(
+            time_frequency=0.002,
+            nodal_results=nodal_results1,
+            gauss_point_results=gauss_point_results1,
+        )
+        json_output_parameters2 = JsonOutputParameters(
+            time_frequency=0.002,
+            nodal_results=nodal_results2,
+            gauss_point_results=gauss_point_results2,
+        )
+
+        # create Load objects and store in the list
+        gid_output_process1 = Output(
+            part_name="test_gid_output",
+            output_dir=r"dir_test",
+            output_name=r"test_gid1",
+            output_parameters=gid_output_parameters1,
+        )
+        gid_output_process2 = Output(
+            part_name="test_gid_output",
+            output_dir=r"dir_test",
+            output_name=r"test_gid2",
+            output_parameters=gid_output_parameters2,
+        )
+        vtk_output_process1 = Output(
+            part_name="test_vtk_output",
+            output_dir=r"dir_test\test_vtk1",
+            output_parameters=vtk_output_parameters1,
+        )
+        vtk_output_process2 = Output(
+            part_name="test_vtk_output",
+            output_dir=r"dir_test\test_vtk2",
+            output_parameters=vtk_output_parameters2,
+        )
+
+        json_output_process1 = Output(
+            part_name="test_json_output1",
+            output_name="test_json_output1",
+            output_dir="dir_test",
+            output_parameters=json_output_parameters1,
+        )
+
+        json_output_process2 = Output(
+            part_name="test_json_output2",
+            output_name="test_json_output2",
+            output_dir="dir_test",
+            output_parameters=json_output_parameters2,
+        )
+        all_outputs = [
+            gid_output_process1,
+            vtk_output_process1,
+            json_output_process1,
+            gid_output_process2,
+            vtk_output_process2,
+            json_output_process2,
+        ]
+
+        # write dictionary for the output(s)
+        kratos_outputs_io = KratosOutputsIO(domain="PorousDomain")
+        test_output = kratos_outputs_io.create_output_process_dictionary(all_outputs)
+
+        # json requires to make the directory, then we remove it...
+        for json_process in test_output["processes"]["json_output"]:
```

Comment:
I don't understand this part, can you explain it? Also is there a danger i accidentally delete the wrong directory?

Reply:
mh perhaps we can just remove this part. In theory it just removes the directory to store the json output when testing locally.

If you think is unnecessary, we can remove it from here.